build-system/tasks/visual-diff/index.js

```diff
@@ -630,7 +630,8 @@ async function ensureOrBuildAmpRuntimeInTestMode_() {
 
 function installPercy_() {
   log('info', 'Running', colors.cyan('yarn'), 'to install Percy...');
-  execOrDie('npx yarn --cwd build-system/tasks/visual-diff');
+  execOrDie('npx yarn --cwd build-system/tasks/visual-diff',
+      {'stdio': 'ignore'});
```

Comment:
nit: prefer splitting minor unrelated edits to separate PRs - it's totally fine to create a PR with 1 line! (but since you've already done this, leave this in)